zio-http-example/src/main/scala/example/APIExamples.scala

```diff
@@ -9,34 +9,35 @@ object APIExamples extends ZIOAppDefault {
   import QueryCodec._
 
   // MiddlewareSpec can be added at the service level as well
-  val getUser =
-    API.get(literal("users") / int).out[Int]
+  val getUserEndpointSpec =
```

Comment:
Calling these `*EndpointSpec` is precise, but does not read as well in the client example:

```scala
getUsersEndpointSpec(...)
```

versus:

```scala
getUsers(...)
```

I suggest that we elide the `EndpointSpec` suffix on these, in order to improve readability; and the fact that they are `EndpointSpec` is already visible in their type signatures.